### PR TITLE
ppc64le: switch virtiofsd from C to rust version

### PR DESCRIPTION
**Pull request by Amulyam24**
We have been using the C version of virtiofsd on ppc64le. Now that the issues with rust virtiofsd have been fixed, let's switch to it.

Fixes: #4259

### CODE REVIEW

**Comment by stevenhorsman** [comment]
@Amulyam24 - I think you might need to rebase since https://github.com/kata-containers/kata-containers/pull/6811 has been merged to get this passing the tests

**Comment by fidencio** [comment]
I have forced-pushed to your branch, bumping the virtiofsd version to v1.6.1.

Let me see if now the cloud-hypervisor CIs will be happy.

**Review by fidencio — APPROVED** [review]
lgtm, thanks @Amulyam24!

The CI issues we hit with Cloud Hypervisor + virtiofsd were actually related to running the tests on an already EOLed Ubuntu.

I've bumped the images in the jenkins side and everything seems to be okay.

Thanks for the work on this one!

**Review by fidencio — APPROVED** [review]
lgtm, thanks @Amulyam24!

The CI issues we hit with Cloud Hypervisor + virtiofsd were actually related to running the tests on an already EOLed Ubuntu.

I've bumped the images in the jenkins side and everything seems to be okay.

Thanks for the work on this one!

File: versions.yaml [hunk 1]
```diff
@@ -297,14 +297,14 @@ externals:
...
     url: "https://gitlab.com/virtio-fs/virtiofsd"
-    version: "v1.3.0"
-    toolchain: "1.62.0"
+    version: "v1.6.1"
+    toolchain: "1.66.0"
     meta:
-      # From https://gitlab.com/virtio-fs/virtiofsd/-/releases/v1.3.0,
-      # this is the link labelled virtiofsd-v1.3.0.zip
+      # From https://gitlab.com/virtio-fs/virtiofsd/-/releases/v1.6.0,
+      # this is the link labelled virtiofsd-v1.6.0.zip
```
Comment (stevenhorsman) [inline versions.yaml:304]: Should these comment links be pointing at v1.6.1 rather than v1.6.0 if that's what we are using?

Comment (fidencio) [reply versions.yaml:304]: Yep, it should! :-)

Comment (fidencio) [reply versions.yaml]: As it was my fault updating the tarball to v1.6.1, but forgetting about the comments, I've updated it.

**Comment by Amulyam24** [comment]
The jobs passed 🎉 going ahead with merging.

**Pull request merged**